### PR TITLE
Add documentation for babel plugins

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

File: packages/babel-plugin-name-amd-modules/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+For example, given a `srcPrefixes` of 
+`["src/main/resources/META-INF/resources"]` if the module under 
+`src/main/resources/META-INF/resources/lib/index.js` is processed, its relative 
+path is computed to `lib/index.js`.
+
+The default value of the `srcPrefixes` option is 
+`["src/main/resources/META-INF/resources"]`.
+
+## Example
```
Comment (jbalsas) [inline packages/babel-plugin-name-amd-modules/README.md]: Can you verify this is being rendered as a title properly? Didn't look like it on the GitHub preview...

File: packages/babel-plugin-name-amd-modules/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+**In**
+
+```
```
Comment (jbalsas) [inline packages/babel-plugin-name-amd-modules/README.md]: All code blocks are missing the language to enable syntax highlighting. Can you review them adding `javascript`, `sh` or `json` where appropriate?

File: packages/babel-plugin-name-amd-modules/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# babel-plugin-name-amd-modules
+
+> Give a name to AMD modules based on package name and version and module path.
+
+## Details
```
Comment (jbalsas) [inline packages/babel-plugin-name-amd-modules/README.md]: Some of the `details` section become too verbose pushing down the more easy to understand examples...

Maybe we could try https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d so at least it shows collapsed on a glimpse through the GitHub interface? Other option would be to push details down, but I'm not sure how much sense does that make.

What do you think?

Comment (izaera) [inline packages/babel-plugin-name-amd-modules/README.md]: I would prefer moving them down and maybe changing the title from Details to Technical Details or something like that. Using <details> sounds good, but it only works with github and that kind of abuses the MD format (I feel).

Comment (jbalsas) [inline packages/babel-plugin-name-amd-modules/README.md]: The `Technical Details` idea sounds good to me 👍

**Pull request merged**